### PR TITLE
make add request design and message design

### PR DESCRIPTION
**Pull request by mrrafi5869**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -48,6 +49,7 @@ function App() {
         <Route path="setting" element={<OrderSettings />} />
         <Route path="send-request" element={<SendRequest />} />
         <Route path="add-request" element={<AddRequest />} />
+        <Route path="message" element={<Message />} />
```
Comment (mahmudulhaquequdrati) [inline src/App.jsx:52]: path should be very clear, ex - orders/request/:id like that

for add request we dont need any new page,

File: src/components/common/AppHead.jsx [hunk 2]
```diff
@@ -75,6 +85,21 @@ export default function AppHead({
             <Button>{addButton.name}</Button>
```
Comment (mahmudulhaquequdrati) [inline src/components/common/AppHead.jsx:85]: we already keep button here, if you add new functionality like showModal or etc, try to do within  <Button>{addButton.name}</Button>

in existing code

File: src/components/common/AppTable.jsx [hunk 3]
```diff
@@ -106,6 +116,8 @@ export function AppTable({
                 <TableRow
                   key={row.id}
                   data-state={row.getIsSelected() && "selected"}
+                  onClick={() => handleRowClick(link)}
+                  className="hover:bg-gray-100 cursor-pointer"
```
Comment (mahmudulhaquequdrati) [inline src/components/common/AppTable.jsx:120]: aikhane row.original mane data object ta

every object a we have id, oikhan teke e navigate kore pathate paren

File: src/pages/order/send-request/AddRequest.jsx
Comment (mahmudulhaquequdrati) [inline src/pages/order/send-request/AddRequest.jsx:1]: amader toh add request page nai,, rather modal ace,, so component use korle e hobe

File: src/components/common/AppTable.jsx (hunk 3 above)
Comment (mrrafi5869) [reply src/components/common/AppTable.jsx:120]: ok

**Pull request merged**